cbrain_task/recon_all/bourreau/recon_all.rb

```diff
@@ -139,7 +147,23 @@ def cluster_commands #:nodoc:
 
     recon_all_command = "recon-all#{lbl_ext} #{with_qcache} #{with_mprage} #{with_3T_data} #{with_cw256} #{with_hippocampal} -sd . #{subjid_info} #{step} #{with_notal_check} #{lbl_options}"
 
+    # Copy license
+    cp_license = <<-CP_LICENSE
+
+# Handle FreeSurfer license
+
+if [ ! -f "$FREESURFER_HOME/license.txt" ] && [ ! -f "$FREESURFER_HOME/.license" ] ; then
```

Comment:
What is that test for `$FREESRUFER_HOME/.license` ? Is this an alternative way to provide a license? And does it have to be a file or a directory? (The -f will only be true if it's a file)

Reply:
It is a file, and yes it is an alternative way to specify a license.